CRM/Batch/Form/Entry.php

```diff
@@ -1242,4 +1249,13 @@ private function standardiseRow(array $row): array {
     return $row;
   }
 
+  /**
+   * Is the current row a renewal.
+   *
+   * @return bool
+   */
+  private function currentRowIsRenew(): bool {
+    return $this->currentRowIsRenewOption === 2;
```

Comment:
I admit this is not a part of the refactoring change but should that options need to be registered in CRM_Core_SelectValues and later 
```
private function currentRowIsRenew(): bool {
   return $this->currentRowIsRenewOption === CRM_Core_SelectValues::membershipModes()['Renew Membership'];
}
```

Reply:
@monishdeb yeah it's been hard enough to figure what it means :-)

Actually the value is distinct to this form I think so we need to make it clear within this class rather than outside the class (I'll keep thinking about ways to do that)